src/cartesian/ReferenceArea.tsx

```diff
@@ -67,7 +67,32 @@ const getRect = (hasX1: boolean, hasX2: boolean, hasY1: boolean, hasY2: boolean,
   return rectWithPoints(p1, p2);
 };
 
-export function ReferenceArea(props: Props) {
+export function ReferenceArea({
+  isFront = false,
```

Comment:
destructuring props here doesn't really do anything for us if they're just added back into the props object below. I guess a future refactor can be to use the destructured values directly where possible.

Reply:
Will work on this and the build failing issue

Reply:
> 

only when am destructuring the xAxis and yAxis are becoming undefined . Any reason why this is happening
![image](https://user-images.githubusercontent.com/54993680/214938886-75952590-cf77-4825-afbe-57a7eb9727b2.png)

Reply:
probably because `0` is falsy, but not sure